test/DynamoCoreWpfTests/NodeViewTests.cs

```diff
@@ -137,5 +139,36 @@ public void ZIndex_NodeAsMemberOfGroup()
                 Assert.AreEqual(2, annotation.ZIndex);
             };
         }
+
+        [Test]
+        public void NodesHaveCorrectLocationsIndpendentOfCulture()
+        {
+            string openPath = @"core\nodeLocationTest.dyn";
```

Comment:
@QilongTang This doesn't need to be appended to the `TestDirectory`?  This probably isn't an issue but just want to make sure the paths are the same on the build machines.  Other than that LGTM

Reply:
@alfarok I had the same question when debugging but found that the `open()` function below will append that in the base class